tests/test_dataset/test_kitti_dataset.py

```diff
@@ -228,52 +204,39 @@ def test_format_results():
     ann_file = 'tests/data/kitti/kitti_infos_train.pkl'
     classes = ['Pedestrian', 'Cyclist', 'Car']
     pts_prefix = 'velodyne_reduced'
-    pipeline = [{
-        'type': 'LoadPointsFromFile',
-        'coord_type': 'LIDAR',
-        'load_dim': 4,
-        'use_dim': 4,
-        'file_client_args': {
-            'backend': 'disk'
-        }
-    }, {
-        'type':
-        'MultiScaleFlipAug3D',
-        'img_scale': (1333, 800),
-        'pts_scale_ratio':
-        1,
-        'flip':
-        False,
-        'transforms': [{
-            'type': 'GlobalRotScaleTrans',
-            'rot_range': [0, 0],
-            'scale_ratio_range': [1.0, 1.0],
-            'translation_std': [0, 0, 0]
-        }, {
-            'type': 'RandomFlip3D'
-        }, {
-            'type': 'PointsRangeFilter',
-            'point_cloud_range': [0, -40, -3, 70.4, 40, 1]
-        }, {
-            'type': 'DefaultFormatBundle3D',
-            'class_names': ['Pedestrian', 'Cyclist', 'Car'],
-            'with_label': False
-        }, {
-            'type': 'Collect3D',
-            'keys': ['points']
-        }]
-    }]
-    modality = {'use_lidar': True, 'use_camera': False}
+    pipeline = [
```

Comment:
It seems that similar pipeline are generated and used multiple times. Can we implement a new function to generate it and reduce code?

Reply:
For sure